api/WebApi.ts

```diff
@@ -423,4 +424,19 @@ export class WebApi {
             return decryptedContent;
         }
     }
+
+    public async getWorkItemTrackingProcessDefinitionApi(serverUrl?: string, handlers?: VsoBaseInterfaces.IRequestHandler[]): Promise<workitemtrackingprocessdefinitionm.IWorkItemTrackingApi> {
+        // TODO: Load RESOURCE_AREA_ID correctly.
+        serverUrl = await this._getResourceAreaUrl(serverUrl || this.serverUrl, "5264459e-e5e0-4bd8-b118-0985e68a4ec5");
```

Comment:
Same resource area id for both? Just want to make sure.

Reply:
Yes, WorkItemTracking, WorkItemTrackingProcess, and WorkItemTrackingProcessDefinitions all have the same area ID